vocode/streaming/models/agent.py

```diff
@@ -83,6 +85,13 @@ class ChatAnthropicAgentConfig(AgentConfig, type=AgentType.CHAT_ANTHROPIC):
     max_tokens_to_sample: int = 200
 
 
+class ActionAgentConfig(AgentConfig, type=AgentType.ACTION):
+    actions: List[ActionType]
+    model_name: str = CHAT_GPT_AGENT_DEFAULT_MODEL_NAME
```

Comment:
should this be `gpt-4`?

Reply:
gpt-4 would be much slower, wouldn't it?